example/case_study.jl

```diff
@@ -53,53 +61,52 @@ h_0(t, κ, λ) = κ/λ * (t/λ)^(κ - 1)
     # ---------------- Priors -----------------
     ## priors longitudinal
     # μ priors, population parameters
-    μ_BSLD ~ LogNormal(3.5,1)
-    μ_d ~ Beta(1,100)
-    μ_g ~ Beta(1,100)
-    μ_φ ~ Beta(2,4)
+    μ_BSLD ~ LogNormal(3.5, 1)
+    μ_d ~ Beta(1, 100)
+    μ_g ~ Beta(1, 100)
+    μ_φ ~ Beta(2, 4)
     μ = (BSLD = μ_BSLD, d = μ_d, g = μ_g, φ = μ_φ)
     # ω priors, mixed/individual effects
-    ω_BSLD ~ LogNormal(0,1)
-    ω_d ~ LogNormal(0,1)
-    ω_g ~ LogNormal(0,1)
-    ω_φ ~ LogNormal(0,1)
+    ω_BSLD ~ LogNormal(0, 1)
+    ω_d ~ LogNormal(0, 1)
+    ω_g ~ LogNormal(0, 1)
+    ω_φ ~ LogNormal(0, 1)
     Ω = (BSLD = ω_BSLD^2, d = ω_d^2, g = ω_g^2, φ = ω_φ^2)
     # multiplicative error
-    σ ~ LogNormal(0,1)
-    ## priors survival
-    κ ~ truncated(LogNormal(0,1),0.8,1.2)
-    λ ~ LogNormal(5,1)
-    
+    σ ~ LogNormal(0, 1)
+    ## prior survival
+    λ ~ LogNormal(5, 1)
+
     ## prior joint model
-    β ~ truncated(Normal(0,0.5),-0.1,0.1)
-    
+    γ ~ truncated(Normal(0, 0.5), -0.1, 0.1)
+
     ## η describing the mixed effects of the population
-    η_BSLD ~ filldist(Normal(0,Ω.BSLD),n)
-    η_d ~ filldist(Normal(0,Ω.d),n)
-    η_g ~ filldist(Normal(0,Ω.g),n)
-    η_φ ~ filldist(Normal(0,Ω.φ),n)
-    η = [(BSLD=η_BSLD[i], d=η_d[i], g=η_g[i], φ=η_φ[i]) for i in 1:n]
+    η_BSLD ~ filldist(Normal(0, Ω.BSLD), n)
+    η_d ~ filldist(Normal(0, Ω.d), n)
+    η_g ~ filldist(Normal(0, Ω.g), n)
+    η_φ ~ filldist(Normal(0, Ω.φ), n)
+    η = [(BSLD = η_BSLD[i], d = η_d[i], g = η_g[i], φ = η_φ[i]) for i = 1:n]
     # Transforms
-    Ψ = [[μ_BSLD * exp(η_BSLD[i]),
-    μ_g * exp(η_g[i]),
-    μ_d * exp(η_d[i]),
-    inverse(logit)(logit(μ_φ) + (η_φ[i]))]
-    for i in 1:n]
+    Ψ = [  [μ_BSLD * exp(η_BSLD[i]),
+            μ_g * exp(η_g[i]),
+            μ_d * exp(η_d[i]),
+            inverse(logit)(logit(μ_φ) + (η_φ[i])),] 
+         for i = 1:n]
 
     # ---------------- Likelihoods -----------------
     # add the likelihood of the longitudinal process
-    for measurement in 1:m
-        id = Int(longit_ids[measurement])
-        meas_time = longit_times[measurement]
+    for data in 1:m
+        id = Int(longit_ids[data])
+        meas_time = longit_times[data]
         sld_prediction = sld(meas_time, Ψ[id], tx)
         if isnan(sld_prediction) || sld_prediction < 0
             sld_prediction = 0
         end
-        longit_measurements[measurement] ~ Normal(sld_prediction, sld_prediction * σ)
+        longit_measurements[data] ~ Normal(sld_prediction, sld_prediction * σ)
     end
     # add the likelihood of the survival model with link
-    baseline_hazard(t) = h_0(t, κ, λ)
-    for i in 1:n
+    baseline_hazard(t) = h_0(t, λ)
+    for i = 1:n
```

Comment:
comment: here you have `i`, so would be good to have something analogous above for the time point indexing